src/bucket/BucketIndexImpl.cpp

```diff
@@ -167,8 +167,18 @@ BucketIndexImpl<IndexT>::BucketIndexImpl(BucketManager& bm,
 
         CLOG_DEBUG(Bucket, "Indexed {} positions in {}",
                    mData.keysToOffset.size(), filename.filename());
+
+        // Our target false positive rate is 0.1% even though we set the bloom
+        // filter false positive rate to 0.05%. We do this because our entry
+        // count estimation can be an underestimation (we assume every
+        // BucketEntry is an account LiveEntry, but TTL and DEADENTRY are
+        // smaller). If we gave a larger entry count estimate, the size of our
+        // bloom filter would significantly increase. Instead, by setting the
+        // desired false positive rate to 0.05%, the bloom filter size stays
+        // approximately the same and we give ourselves an additional 10% of
+        // wiggle room on the estimation.
         if (std::is_same<IndexT, RangeIndex>::value &&
-            estimatedNumElems < count)
+            estimatedNumElems * 1.1 < count)
```

Comment:
Looks like this warning is redundant now that we have Grafana panels to track bloom filter misses.